src/groups.js

```diff
@@ -1,5 +1,9 @@
 import { strict as assert } from 'assert';
+import authzActions from "./authz/actions.js";
+import authzTags from "./authz/tags.js";
```

Comment:
@tsibley

I removed the comment here in the rebase onto ESM:

https://github.com/nextstrain/nextstrain.org/blob/d36f8441e0fd7d9d288cd92aa898caeb356b8a9d/src/authz/index.js#L135-L144

However, circular imports are still an issue. After moving the import back to top-level in 6401dda32e25de051d7b77323325fdc2439aef24, tests still fail due to circular imports. This one is easy to fix though, since a more selective import isn't circular anymore: c2fe5d84e21a659143ae498af3bd5f3b2a2a4499

Does this seem reasonable to you?

Reply:
I suppose? I think it's unfortunate that we have to, e.g., directly use `authzTags` and `authz/tags.js` instead of delegating via `authz.tags` since restructurings will require updating the callsites, but it's a minor thing in the big picture.

Reply:
Nevermind, this workaround isn't needed after upgrading Jest. See https://github.com/nextstrain/nextstrain.org/pull/570#discussion_r977050514